File: rest/message/list-get-example-1/list-get-example-1.5.x.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'twilio-ruby'

# Get your Account SID and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'

# Initialize Twilio Client
@client = Twilio::REST::Client.new(account_sid, auth_token)

# Get message with the given sid
@message = @client.api.messages('MMXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX').fetch

@client.messages.list.each do |message|
  puts message.body
end

File: rest/member/list-get-example-1/list-get-example-1.5.x.cs
// Download the twilio-csharp library from twilio.com/docs/csharp/install
using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account.Queue;

class Example
{
  static void Main(string[] args)
  {
    // Find your Account Sid and Auth Token at twilio.com/console
    const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
    const string authToken = "your_auth_token";
    TwilioClient.Init(accountSid, authToken);

    var members = MemberResource.Read("QU5ef8732a3c49700934481addd5ce1659");

    foreach (var member in members)
    {
      Console.WriteLine(member.WaitTime);
    }
  }
}
